### ISSUE TITLE
Re-add manual example to force password manager

### ISSUE THREAD

**Issue by Ambrevar** [atlas-engineer/nyxt#1147]
In #1144 the example for force-setting the password manager to, say, KeepassXC when `pass` was also installed is gone.

We need to add an example showing how to set a specific password manager.

**Comment by jmercouris** [comment]
Should we talk about how you could just mutate the `password:*interfaces*` variable, or do you have something else in mind?

**Comment by Ambrevar** [comment]
Shouldn't we use `define-configuration` instead?

Is the following correct?

```lisp
(define-configuration (browser)
    ((password-interface (make-instance 'password:user-keepassxc-interface))))
```

**Comment by Ambrevar** [comment]
Fixed with 0b78bcff0d313bb93d5de4440eccd30ac9a3e560.

**Comment by Ambrevar** [comment]
Reopening, because I forget to address the above point: should we use `define-configuration` instead?

Would be better I think because less global and more consistent.

However we can't right now because `password-interface` is unexported.  Is there a good reason for this?  Seems limiting, I'd rather export it.

@jmercouris?

**Comment by jmercouris** [comment]
Why not :-)

**Comment by Ambrevar** [comment]
I've just sent a fix in #1177.  Let me know what you think.
